src/content/docs/integrations/mlops-integrations/dagshub-mlops-integration.mdx

```diff
@@ -0,0 +1,48 @@
+
+MLOps provides a tool for monitoring and observing the performance and effectiveness of machine-learning models in a production environment. This increases the possibilities for collaboration between data science and DevOps teams, feeding into a continuous process of development, testing, and operational monitoring.
+
+## The DagsHub integration [#dagshub]
+
+DagsHub is a platform for data scientists and machine learning engineers to version and sync their data, models, experiments, and code. It allows you and your team to easily share, review and reuse your work, providing a GitHub-like experience for machine learning. DagsHub is built on popular open-source tools and formats, making it easy to integrate with the tools you already use like New Relic.
+
+New Relic now offers an integration with DagsHub to monitor and analyze your machine learning training metrics in real-time. 
+
+## Integrate DagsHub with New Relic [#integrate]
+
+Integrating DagsHub and New Relic enables you to analyze and monitor machine learning training metrics in real-time. You can visualize your metrics in a [New Relic dashboard](/docs/query-your-data/explore-query-data/dashboards/introduction-dashboards/), create custom metrics, and set [alerts](/docs/alerts-applied-intelligence/new-relic-alerts/learn-alerts/introduction-alerts/) to notify you of events and incidents happening during your training runs.
+
+In order to receive data from DagsHub into New Relic, you need your account number and a special key which you can obtain as follows:
+
+* **Log into your New Relic account**: Go to [one.newrelic.com](https://one.newrelic.com), and click **Explorer**. On the upper hand right corner, on the main navigation menu, click on **+Add more data**.
+* Click on DagsHub: Type DagsHub in the search bar, or scroll down to the MLOps Integration section, and click the **DagsHub** icon.
+* **Select the account ID** you want DagsHub to integrate with.
+* **Create an access token**: Once you've selected an account ID, under real-time insert metrics, click **Create an API key** . This will be your new telemetry API key. Keep the New Relic page open for future steps. 
+* Log into the **DagsHub portal** and select the repository you're working on. Click **Settings**, and select the integrations button from the left hand menu. Click on the **New Relic** tile. 
+* **Copy and paste the token in DagsHub**. Go back to the New Relic integration dashboard and copy the token you created by clicking on the **copy** icon next to the insert key. On the DagsHub’s portal, paste the insert key under **New Relic Insight insert key**, and finish by clicking **Next**. The token will be verified, and a confirmation screen will appear.
+
```

Comment:
Maybe we can number these steps instead of being bullet points? To give it more of a procedure feel?